content/modules/ROOT/pages/m4/module-04.5.adoc

```diff
@@ -76,25 +76,25 @@ image:m4/parasol-branch-first-pipeline-success.png[]
 
 [NOTE]
 ====
-Login to OpenShift using the `htpasswd` provider and the credentials `admin` / `{common_password}`.
```

Comment:
@btison @jayachristina for some reason this wasn't presented to me in my environment, but provisioning the environment using the Summit specific item in demo-dot *does* show the _htpasswd_ and _kube:admin_ options. I guess we should leave this instruction to be safe.